### ISSUE TITLE
Broken Link

### ISSUE THREAD

**Issue by void-hr** [keshavgbpecdelhi/Web-Development#121]
**Describe the bug**
Demo Link of Roll Dice is broken

**To Reproduce**
1. Go to '[README.md](https://github.com/keshavgbpecdelhi/Web-Development#readme)'
2. Click on '[Roll The Dice Demo](https://github.com/keshavgbpecdelhi/Web-Development/blob/master)'
4. [See error](https://github.com/keshavgbpecdelhi/Web-Development/blob/master)

**Comment by keshavgbpecdelhi** [comment]
Yes, I was about to do it. Could you edit it with : https://keshavgbpecdelhi.github.io/Web-Development/Roll-the-Dice 
And make sure other links also direct correct link !

**Comment by void-hr** [comment]
Done! Thank you for your support

**Comment by keshavgbpecdelhi** [comment]
Closing wrt #122
